addons/web/static/src/core/barcode/barcode_video_scanner.js

```diff
@@ -23,6 +23,7 @@ export class BarcodeVideoScanner extends Component {
         close: { type: Function, optional: true },
         onResult: Function,
         onError: Function,
+        placeholder: { type: String, optional: true },
```

Comment:
remove?

Reply:
We need to keep this, `BarcodeVideoScanner` actually receives placeholder in props.